src/main/java/Wolox/training/models/User.java

```diff
@@ -40,6 +43,7 @@ public long getId() {
     }
 
     public void setId(long id) {
+        checkNotNull(id);
         this.id = id;
     }
```

Comment:
This method should be removed, because only JPA/Hibernate can set the Id through [Java Reflections](https://www.oracle.com/technical-resources/articles/java/javareflection.html)